activiti-services/activiti-services-core-model/src/main/java/org/activiti/services/core/model/ProcessDefinitionMeta.java

```diff
@@ -14,14 +14,14 @@
     private String name;
     private String description;
     private int version;
-    private List<String> users;
-    private List<String> groups;
+    private HashSet<String> users;
```

Comment:
@gmalanga Please change this to Set, the interface instead of the HashSet implementation here. 
The same applies to all the methods.